src/domains/wallinDomain.cpp

```diff
@@ -31,178 +31,168 @@
 
 namespace ghost
 {
-  WallinDomain::WallinDomain( int col,
-			      int row,
-			      int nbVar,
-			      int sRow,
-			      int sCol,
-			      int tRow,
-			      int tCol ) 
-    : Domain(col*row+1, nbVar, -1),
-      mCol_(col),
-      nRow_(row),
-      matrixType_(vector< vector<string> >(nRow_, vector<string>(mCol_, "") ) ),
-      matrixId_(vector< vector< set<int> > >(nRow_, vector< set<int> >(mCol_, set<int>() ) ) ),
-      startingTile( make_pair( sRow, sCol ) ),
-      targetTile( make_pair( tRow, tCol ) )
-  { 
-    matrixType_[sRow][sCol] += "@s";
-    matrixType_[tRow][tCol] += "@t";
-  }
+	WallinDomain::WallinDomain(int maxX, int maxY, int nbVar, int sX, int sY, int tX, int tY)
+		: Domain(maxX*maxY + 1, nbVar, -1),
+		maxX_(maxX),
+		maxY_(maxY),
+		matrixType_(vector< vector<string> >(maxY, vector<string>(maxY, ""))),
+		matrixId_(vector< vector< set<int> > >(maxY, vector< set<int> >(maxY, set<int>()))),
+		startingTile(make_pair(sX, sY)),
+		targetTile(make_pair(tX, tY))
+	{
+		matrixType_[sX][sY] += "@s";
+		matrixType_[tX][tY] += "@t";
+	}
 
-  WallinDomain::WallinDomain( int col,
-			      int row,
-			      const vector< pair<int, int> > &unbuildables,
-			      const vector< Building > *variables,
-			      int sRow,
-			      int sCol,
-			      int tRow,
-			      int tCol ) 
-    : WallinDomain( col, row, variables->size(), sRow, sCol, tRow, tCol )
-  {
-    for( const auto &u : unbuildables )
-      matrixType_[u.first][u.second].assign(3, '#');
+	WallinDomain::WallinDomain(int maxX, int maxY,
+		const vector< pair<int, int> > &unbuildables, const vector< Building > *variables,
+		int sX, int sY, int tX, int tY)
+		: WallinDomain(maxX, maxY, variables->size(), sX, sY, tX, tY)
+	{
+		for (const auto &u : unbuildables) {
```

Comment:
Please, keep the same space and bracket policy as above, like 
for( const auto &u : unbuildables )
{
  blabla
}